cmd/konvoy-image/cmd/validate.go

```diff
@@ -0,0 +1,54 @@
+package cmd
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/mesosphere/konvoy-image-builder/pkg/app"
+)
+
+var validateFlags app.ValidateFlags
+
+// validateCmd runs validations against nodes to provision.
+var validateCmd = &cobra.Command{
+	SilenceUsage:  true,
+	SilenceErrors: true,
+	Use:           "validate",
+	Short:         "validate existing infrastructure",
+	Args:          cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if !isValidCIDR(validateFlags.PodSubnet) {
+			return fmt.Errorf("pod-subnet %q was not a valid CIDR", validateFlags.PodSubnet)
+		}
+
+		if !isValidCIDR(validateFlags.ServiceSubnet) {
+			return fmt.Errorf("service-subnet %q was not a valid CIDR", validateFlags.ServiceSubnet)
+		}
+
+		if err := app.Validate(args[0], validateFlags); err != nil {
```

Comment:
Instead of taking in the `inventory.yaml` as an arg, lets be consistent with https://github.com/mesosphere/konvoy-image-builder/blob/43b64691036e379d1b70fbeb4ff5404b38d1cbf2/cmd/konvoy-image/cmd/provision.go#L42

And make it a flag please.

Reply:
This was a change done recently today, so thats my excuse for not catching this :p

Reply:
actually there were a lot of changes to the provision.go file. it is not longer used by cmd.